bolt/lib/Rewrite/LinuxKernelRewriter.cpp

```diff
@@ -207,8 +218,6 @@ void LinuxKernelRewriter::insertLKMarker(uint64_t PC, uint64_t SectionOffset,
 }
 
 void LinuxKernelRewriter::processLKSections() {
-  assert(BC.IsLinuxKernel && "Linux kernel binary expected.");
```

Comment:
How come are you removing this assertion?

Reply:
This function used to be under `RewriteInstance`. Since I moved it some time ago under `LinuxKernelRewriter` it became obsolete. Can put in a separate PR.